redhat/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM registry.access.redhat.com/rhel
+
+MAINTAINER Kong
+
+ENV KONG_VERSION 1.0.0rc2
```

Comment:
We should probably start with releasing 0.14 first? Nothing prevents us from providing 1.0.0rc2 subsequently.

Reply:
Agreed.